lib/logstash/outputs/elasticsearch/elasticsearch-template.json

```diff
@@ -5,88 +5,37 @@
   },
   "mappings" : {
     "_default_" : {
-      "_all" : {"enabled" : true, "omit_norms" : true},
```

Comment:
Hmm this must be a bug. :( I just tried locally and it worked for me, do you have steps that I can follow to reproduce?